dev/tasks/java-jars/github.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          - { runs_on: ["macos-latest"], arch: "x86_64"}
+          - { runs_on: ["macos-13"], arch: "x86_64"}
```

Comment:
macos-latest == macos-14 now so if we want amd64 we have to explicitly request macos-13